docs/development/getting-started.rst

```diff
@@ -401,6 +401,23 @@ away your builds + ``docker`` containers and start again:
 ``docker volume rm $(docker volume ls -q --filter dangling=true)``
 
 
+Compilation errors in non-Docker development
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+While Warehouse is designed to be developed in using Docker in certain
+situations you may be tempted to install Warehouse's requirements in your
```

Comment:
Tempted?

Reply:
I guess it's too late at this point if they're reading the troubleshooting section 😄 I'll rephrase